Tests/RxSwiftTests/Observable+MergeTests.swift

```diff
@@ -1222,7 +1222,7 @@ extension ObservableMergeTest {
     func testFlatMapFirst_Complete() {
         let scheduler = TestScheduler(initialClock: 0)
 
-        let xs = scheduler.createHotObservable([
+        let xs: TestableObservable<TestableObservable<Int>> = scheduler.createHotObservable([
```

Comment:
Why is this needed? Seems to work without.

Reply:
This line cause type-check time limit warning:
```
/Users/travis/build/ReactiveX/RxSwift/Tests/RxSwiftTests/Observable+MergeTests.swift:1225:28: expression took 155ms to type-check (limit: 100ms)
```
This fix prevents it. If u think this warning is ok, i am ready to discard changes and leave only deprecation fix + swiftlint disabling